CHANGELOG

```diff
@@ -1,4 +1,5 @@
 v3.1 (2022-11-4)
+
```

Comment:
Since this is no longer markdown, I prefer to not have the spaces between the release and the changes. Please revert this and the rest.

Reply:
You got it, sorry for missing that!